### DISCUSSION TITLE
implement boostrap in a primevue project without typescript

### DISCUSSION THREAD

**Discussion by xkiller838** [primefaces/community#1129] [PrimeVue] [answered]
Hi, I have a question about how I can implement boostrap 4 in a primevue project without typescript. The examples provided on the page are mostly typescript, not normal javascript. In addition, I would like to know if there is a way to not have to import all the components in the main.js so that they work and only import the ones I need in the component. For example, if I need a slider in a component, just import that without having to import everything. I don't know if it is possible. The documentation and examples of use of boostrap are very simple. to understand I appreciate any help

**Comment by tugcekucukoglu** [comment, marked as answer]
You do not need to import all the components, you can only import the ones you want to use. If you are using Nuxt, you can try the 'nuxt-primevue' module. If you are using bootstrap, we recommend that you examine other pages as well because PrimeVue will work the same for all themes.

**Reply by xkiller838** [reply]
I am using boostrap 5.2 in vue 3 and I am trying to follow the example that is in boostrap on the primevue documentation page but the examples are very simple buttons one table above but the modal one is not complete and the examples are very few, I would appreciate adding some more examples of how to implement boostrap in the modal and other components, not just buttons and a table, I tried to follow the same example using pt that is in the table in the Diaolog modal but it did not work for me
